dockerfiles/idp_deploy.Dockerfile

```diff
@@ -208,6 +208,7 @@ WORKDIR $RAILS_ROOT
 # set bundler up
 RUN bundle config build.nokogiri --use-system-libraries
 RUN bundle config set --local deployment 'true'
+RUN bundle config set --local deployment 'true'
```

Comment:
seems duplicative?

Reply:
Just a quick test, having some issues with container building that might be caching related?

Reply:
I have learned it is due to an updated gem now including a folder that has spaces in the folder name

Reply:
Hopefully fixed in [38bc641](https://github.com/18F/identity-idp/pull/11856/commits/38bc641ec6d261dfb870a6807a4fa65a8dd7a512)